runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerBarOverlay.java

```diff
@@ -170,4 +208,4 @@ void onTick()
 			showingPrayerBar = false;
 		}
 	}
-}
+}
```

Comment:
(I'll commit this via GitHub's UI)
```suggestion
}

```